apps/sports-helsinki/src/domain/search/combinedSearch/search.module.scss

```diff
@@ -98,26 +98,6 @@ $buttonWidth: 180px;
   }
 }
 
-.autoSuggestRowWithTargetGroupFilter {
-  @extend .autoSuggestRowBase;
```

Comment:
It would be good to remove `.autoSuggestRowBase` also completely as it is now left unused by your changes.